plugins/sensors/switchbotmeter

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+  switchbotmeter - Munin plugin to monitor temperature and humidity with SwitchBot Meter
+
+=head1 CONFIGURATION
+
+  Obtain a token and deviceid for SwitchBot API. See the following website.
+  https://github.com/OpenWonderLabs/SwitchBotAPI
+
+=head1 ENVIRONMENT VARIABLES
+
+  env.token    : Token to access SwitchBot API. (Required)
+  env.deviceid : Device ID of SwitchBot Meter. (Required)
+  env.interval : Interval in seconds for API access. (Optional, default is 0)
+  env.tempunit : Temperature unit. (Optional, default is C)
+
+=head1 NOTES
+
+  The amount of SwitchBot API calls per day is limited to 1000 times.
+  If munin-node executes this plugin every 5 minutes, the API will be called 288 times a day.
+  You can use env.interval parameter to prevent frequent API access. For example, if you set
+  env.interval 900, the API response will be cached to local file for 900 seconds(15 minutes).
+
+=head1 AUTHOR
+
+  K.Cima https://github.com/shakemid
+
+=head1 LICENSE
+
+  GPLv2
+
```

Comment:
Maybe you want to add an SPDX identifier:
```
SPDX-License-Identifier: GPL-2.0-only
```

Reply:
All right. I will add it.